src/bundle/Resources/public/scss/_card.scss

```diff
@@ -17,11 +17,10 @@
     &__header {
         background-color: $ez-color-base-pale;
         color: $ez-black;
-        font-weight: 700;
         border-radius: .25rem .25rem 0 0;
         padding: .75rem 1.25rem .5rem 1.25rem;
 
-        &--secondary {
```

Comment:
This will have an impact on content type create, content type edit, role assignment create. Do we want it in all places?
What I mean is class `ez-card__header--secondary` is used in few places. Do we want to not style it anymore?

Reply:
And I don't understand why we need this class `ez-card__title` and we cannot use `ez-card__header--secondary`? This should work the same as the previous solution, in content type edit.

Reply:
Updating it. I see what you mean. I am ok keeping it.